int16 angle1
int16 speed1
int16 angle2
int16 speed2
